components/brave_rewards/resources/page/components/connect_wallet_modal.tsx

```diff
@@ -126,26 +117,6 @@ export function ConnectWalletModal (props: Props) {
             <style.panelText>
               {getString('connectWalletChooseText')}
             </style.panelText>
-            {
-              showMinimumBalanceWarning && selectedProvider &&
-                <style.minimumBalanceWarning>
-                  {
-                    formatMessage(getString('minimumBalanceWarning'), [
-                      selectedProvider.name,
-                      getMinimumBalance(selectedProvider.type)
-                    ])
-                  }
-                  <div className='login'>
-                    <a
-                      data-test-id='connect-login-button'
```

Comment:
I think this element is referenced from a Rewards browser test. We can probably remove that test now. (`UpholdLimitNoBAT` maybe?)